offchain/rollups-events/Cargo.toml

```diff
@@ -3,14 +3,27 @@ name = "rollups-events"
 version = "1.0.0"
 edition = "2021"
 
+[lib]
+path="src/lib.rs"
+
+[[bin]]
+name="broker-tls-test"
+path="src/main.rs"
```

Comment:
It is strange for this file to be called `main.rs`. Maybe rename it to `broker_tls_test.rs`?